### DISCUSSION TITLE
Is work on TsMuxer still going on?

### DISCUSSION THREAD

**Discussion by Nemesh64** [justdan96/tsMuxer#903] [General]
The last update was 0n 2024-06-06.
More then 4 months ago.
Is work on TsMuxer still going on?
With regards,

Nemesh

**Comment by justdan96** [comment]
Work has slowed down, but we are always looking for new contributors!

**Reply by donnafaye0323** [reply]
I haven't heard from you for a short time. Will this project be open-source? It will have new updates soon. I love this tool, and I would help it out some more.

**Comment by Nemesh64** [comment]
Dan, I whis I was able to help you programming.
In the past (going back to the 80's) I was able to program in BASIC (ZX81) en type over the listings
in the magzine 'Your Computer'. (Yes I'am old).
Started with a 1KB ZX81 and later expaneded it to 16KB (Now working with a PC with 128GB)
This one: https://nl.wikipedia.org/wiki/Sinclair_ZX81
Tried to program in machincecode but I was not able to do that.
I know some bugs reported on the issu's page require a tremdous amount of programming work.
Never the less, thanks for the reply and whis you all the best.

**Comment by donnafaye0323** [comment]
This year, will there still be work to do with updates? I will hear it from you soon.

**Comment by Nemesh64** [comment]
Hello Donna,

Justdan96 and others that help creating and fixing bugs in TsMuxer don't get payed for their work.
They do everything in their free time.
Justdan96 has improved TsMuxer from version 2.6.12 and repaired a lot of bugs in TsMuxer.
There are still bugs in the software, but (as I have understood from Justdan96) it requires many, many hours of programming to squash those bugs.
Thats why he replied to me: "but we are always looking for new contributors!"
I use TsMuxer on a daily basis many times and so it's a very important tool for me.
My biggest fear is that TsMuxer will become abandonware and that we will have to do it with the currend version from 6 June 2024.
Never the less, a big thanks to Justdan96 and others that helped him to improve TsMuxer.
With regards,

Nemesh
